docs/API/LorisRESTAPI_v0.0.3.md

```diff
@@ -814,3 +814,49 @@ files, and a `.tar.gz` of the raw DICOM data as acquired during the candidate
 scanning session, and as retrieved from `/candidates/$CandID/$Visit/dicoms`.
 
 Only `GET` is currently supported.
+
+### 5.3 Upload DICOM file
+```
+PUT /candidates/$CandID/$VisitLabel/dicoms/$Filename/$IsPhantom
+```
+
+Upload a DICOM file that is compressed as a `.zip`,`.tar.gz` or `.tgz`.
+$CandID must be the full name of the file and its extension.
+$IsPhantom must be `'Y'` or `'y'` if it's a phantom file, else `'N'` or `'n'` 
```

Comment:
Could the isPhantom be derived from the entity_type of the candidate? Usually, when the candidate entity_type is `scanner` the images are phantoms. @cmadjar right?

Reply:
indeed, given that the CandID is in the URL, you technically could derive whether the scan is a phantom by looking at the "Entity_type" of the candidate (would be "Scanner" for phantom datasets or "Human" for actual candidates.

At least, this is the current behaviour. Hope this helps!

Reply:
Is the value of 'Entity_type' 'Scanner' for human phantoms?